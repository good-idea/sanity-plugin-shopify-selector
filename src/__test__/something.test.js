/* eslint-disable no-undef */
import React from 'react'
import { render } from 'react-testing-library'

// import Component from '../Component'

describe('Component Component', () => {
	it.only('Mounts Correctly', () => {
	/* Arrange */
		const { container } = render(<div />)
	/* Act */
	
	/* Assert */
		expect(container).toMatchSnapshot()
	})
})
