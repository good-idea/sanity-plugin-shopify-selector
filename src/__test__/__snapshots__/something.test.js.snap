// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Component Mounts Correctly 1`] = `
<div>
  <span />
</div>
`;
